Merit_Visualization.js

```diff
@@ -0,0 +1,304 @@
+//////////////////////////////////////////////////////////////
```

Comment:
please add a copyright statement at the top 
(eg, see https://github.com/google/earthengine-community/pull/200/files)